File: aspnet-core/src/HRMv2.Core/Manager/ChartDetails/Dto/ChartDetailFullDto.cs
using Abp.AutoMapper;
using NccCore.Anotations;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HRMv2.Manager.ChartDetails.Dto
{
    [AutoMap(typeof(ChartDetailDto))]
    public class ChartDetailFullDto : ChartDetailSelectionDto
    {
        public long Id { get; set; }

        public long ChartId { get; set; }

        public string Name { get; set; }

        public string Color { get; set; }

        public bool IsActive { get; set; }
    }
}

File: angular/src/app/service/model/chart-settings/chart-detail-settings/chart-detail-full.dto.ts
import { ChartDetailSelectionDto } from "./chart-detail-selection.dto";

export class ChartDetailFullDto extends ChartDetailSelectionDto {
  id: number;
  chartId: number;
  name: string;
  color: string;
  isActive: number;
  chartDataType?: number;
  isViewOnly: boolean = false;
}
